siliconcompiler/report/dashboard/cli/board.py

```
@@ -932,15 +932,18 @@ def _render(self):
         def update_data():
             try:
                 self._update_rendable_data()
-            except:  # noqa E722
-                # Catch any multiprocessing errors
+            except Exception:
+                # The render thread reads manager proxies, which raise once the
+                # manager goes away during teardown. Nothing is logged here on
+                # purpose: this module is the sink the logger writes into, so a log
+                # call from a dead proxy would raise inside its own handler.
                 pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)